### PR TITLE
Fixes for ALMA bugs from #1664

### PR DESCRIPTION
**Pull request by keflavich**
WIP

* [x] table concatenation
* [ ] Login pages downloaded instead of files

### CODE REVIEW

**Comment by bsipocz** [comment]
If you want to add the PR number to the changelog entry of the previous PR, otherwise I don't think a new entry is necessary.

**Comment by bsipocz** [comment]
please do rebase, I'm not sure why appveyor got triggered, but in either case this is based on an outdated commit.

**Comment by bsipocz** [comment]
CI failure is related, please fix it.

File: astroquery/query.py [hunk 1]
```diff
@@ -59,11 +59,12 @@ def timeout(self, value):
             self._timeout = value
 
     def request(self, session, cache_location=None, stream=False,
-                auth=None, verify=True):
+                auth=None, verify=True, allow_redirects=True):
```
Comment (bsipocz) [inline astroquery/query.py]: Changes to this file messes up the tests for the rest of the modules. Could you double check?

Comment (keflavich) [reply astroquery/query.py]: OK weird, I don't see why that would...

**Comment by keflavich** [comment]
alright, probably time to merge this and open the next bugfix branch...

**Pull request merged**